res/openvpn-gui-res-no.rc

```diff
@@ -222,34 +222,42 @@ BEGIN
     IDS_TIP_CONNECTED_SINCE "\nTilkoblet siden: "
     IDS_TIP_ASSIGNED_IP "\nTildelt IP: %s"
     IDS_MENU_SERVICE "OpenVPN Service"
+    IDS_MENU_IMPORT "Importer fil…"
     IDS_MENU_SETTINGS "Innstillinger…"
     IDS_MENU_CLOSE "Avslutte"
     IDS_MENU_CONNECT "Koble til"
     IDS_MENU_DISCONNECT "Koble fra"
-    IDS_MENU_STATUS "Vis Status"
-    IDS_MENU_VIEWLOG "Vis Logg"
+    IDS_MENU_STATUS "Vis status"
+    IDS_MENU_VIEWLOG "Vis logg"
     IDS_MENU_EDITCONFIG "Endre innstillinger"
     IDS_MENU_PASSPHRASE "Endre passord"
-    IDS_MENU_SERVICE_START "Starte"
-    IDS_MENU_SERVICE_STOP "Stoppe"
-    IDS_MENU_SERVICE_RESTART "Starte på nytt"
+    IDS_MENU_CLEARPASS "Glem lagrede passord"
+    IDS_MENU_SERVICE_START "Start"
+    IDS_MENU_SERVICE_STOP "Stopp"
+    IDS_MENU_SERVICE_RESTART "Start på nytt"
     IDS_MENU_SERVICEONLY_START "Koble til"
     IDS_MENU_SERVICEONLY_STOP "Koble fra"
     IDS_MENU_SERVICEONLY_RESTART "Koble til på nytt"
-    IDS_MENU_ASK_STOP_SERVICE "Vil du koble fra? (Stoppe OpenVPN Service)?"
+    IDS_MENU_ASK_STOP_SERVICE "Vil du koble fra (stoppe OpenVPN Service)?"
 
     /* Logviewer - Resources */
-    IDS_ERR_START_LOG_VIEWER "Feil under oppstart av log viewer: %s"
-    IDS_ERR_START_CONF_EDITOR "Feil under oppstart av konfig editor: %s"
+    IDS_ERR_START_LOG_VIEWER "Feil under oppstart av program for loggvisning: %s"
+    IDS_ERR_START_CONF_EDITOR "Feil under oppstart av konfigurasjons-editor: %s"
 
     /* OpenVPN */
-    IDS_ERR_MANY_CONFIGS "Du kan ikke ha fler enn %d konfig-filer. Kontakt utvikleren av OpenVPN GUI om du har behov for å håndtere fler."
-    IDS_ERR_ONE_CONN_OLD_VER "Du kan bara ha en tilkobling igang samtidig med eldre versjoner av OpenVPN enn 2.0-beta6."
+    IDS_ERR_MANY_CONFIGS "Du kan ikke ha flere enn %d konfigurasjonsfiler. Kontakt utvikleren av OpenVPN GUI om du har behov for å håndtere flere filer."
+    IDS_NFO_NO_CONFIGS "Ingen lesbare tilkoblingsprofiler (konfigurasjonsfiler) funnet"
```

Comment:
Introduced a new term here "tilkoblingsprofiler". Should probably either be used more places as well, or not used at all. The English translation reads "No readable connection profiles (config files) found"